website/docs/configuration/securing-access-to-the-dashboard.mdx

```diff
@@ -54,16 +11,20 @@ There are 2 supported methods for logging in to the dashboard:
 
 The recommended method is to integrate with an OIDC provider, as this will let you control permissions for existing users and groups that have already been configured to use OIDC. However, it is also possible to use a cluster user account to login, if an OIDC provider is not available to use. Both methods work with standard Kubernetes RBAC.
 
-:::note FEATURE TOGGLE 
+:::note FEATURE TOGGLE
```

Comment:
This isn't behind a feature flag.

In fact, this page is a bit funny now that half of this has been moved to the quickstart. I wonder if this should be re-written to be an "Enabling SSO" page :thinking: